content/en/docs/ambient/usage/extend-waypoint-wasm/test.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2154
+
+# Copyright 2023 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# @setup profile=none
+
+set -e
+set -u
+set -o pipefail
+
+source "content/en/docs/ambient/getting-started/snips.sh"
+
+# Kubernetes Gateway API CRDs are required by waypoint proxy.
+snip_download_and_install_2
+
+# install istio with ambient profile
+snip_download_and_install_3
+
+_wait_for_deployment istio-system istiod
+_wait_for_daemonset istio-system ztunnel
+_wait_for_daemonset istio-system istio-cni-node
+
+_verify_like snip_download_and_install_5 "$snip_download_and_install_5_out"
+
+# deploy test application
+snip_deploy_the_sample_application_1
+snip_deploy_the_sample_application_2
+
+snip_deploy_the_sample_application_3
+snip_deploy_the_sample_application_4
+
+# test traffic before ambient mode is enabled
+_verify_contains snip_verify_traffic_sleep_to_ingress "$snip_verify_traffic_sleep_to_ingress_out"
```

Comment:
i do not see any of these verifications in the actual document.. do we need them?

Reply:
it is needed in the getting started document for ease of understanding, but in the wasm document i do not think we have to do this verification

Reply:
Ok, I'll remove the verifications and add sample applications directly to the Ambient Mesh section